schematics/nav/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.html

```diff
@@ -0,0 +1,18 @@
+<mat-drawer-container>
+  <mat-drawer mode="side" #drawer [opened]="!isHandset">
+    <mat-toolbar color="primary">Menu</mat-toolbar>
```

Comment:
Does the sidenav need a toolbar / header? Not sure if I typically see this.

Reply:
I have seen this on Google products before, they will often put the product logo in a toolbar in the sidenav so it lines ups with the main toolbar when the sidenav opens. Google photos and play music are currently doing this, but it seems like most products are switching to the inbox style where the sidenav just opens below the toolbar

Reply:
@jelbourn - Let me know what you'd like me to do here.